### ISSUE TITLE
system tray icon momentarily expands out of folded(grouped) tray menu and goes back to folded tray flashing 

### ISSUE THREAD

**Issue by inuonly** [subhra74/xdm#1117]
### XDM is currently being re-written to be more user friendly and performant, please refer to (https://github.com/subhra74/xdm/discussions/768) for more information ###

**PLEASE DO NOT JUST SAY "It does not work, or something not working etc." Provide enough relevent details so that the issue can be analyzed and reproduced easily**

**Describe the bug**
The system tray icon is moved to the folded menu but it keeps popping out to main system tray and goes back to folded menu very fast like flashing the icon momentarily. 
Verison is 8.0.29
**To Reproduce**
1. Install XDM 8.0.29
2. move the system tray icon to folded colection of all tray icons
3. Now observer system tray , and xdm tray icon keeps xdm icon pop off the folded menu and then makes it go back to folded like flashing a blue blob very fast. The moment you take the icon out of minimised group and on main system tray the behaviour disappers. Only when you move icon inside minimised group of system tray icons then it starts behaving the same way. 

**Expected behavior**
Once added to minimsed system tray group, the icon should stay there and not flash outside. 

**Screenshots**
Since it flashes very fast I am still trying to show where it shows up 
![image](https://github.com/subhra74/xdm/assets/26686197/cbc8c26e-6bc9-48ab-af38-8e111520095c)

**please complete the following information:**
 - OS: Windows 10 Pro
 - Browser Firefox+Chrome (both extensions installed) 
 - XDM addon Version XDM v8+ Browser Helper version 3.4 firefox
 - XDM Version  8.0.29

**Generated log using below method**
 - https://github.com/subhra74/xdm/wiki/Generate-log-for-troubleshooting

**Additional context**

**Comment by inuonly** [comment]
I finally managed to get it recorded . Here is my diagnosis on the issue. 

I recorded system tray right when this issue was occuring and it occured for a long time giving me window to capture the event. 


I am attaching the video showing the real time appearance of the event in system tray. 


I am also sharing the discection of the video in editor to show the frame and the frame itself as follows to show it is indeed xdm icon and it momentarily shows as expanded and within 1 frame it goes back to closeed tray. This keeps resizing icons on taskbar. 
Along with this at the time of recording i am also sharing the version adn screen shot showing it is updated to latest version available. 

> ### XDM is currently being re-written to be more user friendly and performant, please refer to (#768) for more information
> 
> **PLEASE DO NOT JUST SAY "It does not work, or something not working etc." Provide enough relevent details so that the issue can be analyzed and reproduced easily**
> 
> **Describe the bug** The system tray icon is moved to the folded menu but it keeps popping out to main system tray and goes back to folded menu very fast like flashing the icon momentarily. Verison is 8.0.29 **To Reproduce** Steps to reproduce the behavior:
> 
>     1. Install XDM 8.0.29
> 
>     2. move the system tray icon to folded colection of all tray icons
> 
>     3. Now observer system tray , and xdm tray icon keeps xdm icon pop off the folded menu and then makes it go back to folded like flashing a blue blob very fast. The moment you take the icon out of minimised group and on main system tray the behaviour disappers. Only when you move icon inside minimised group of system tray icons then it starts behaving the same way.
> 
> 
> **Expected behavior** Once added to minimsed system tray group, the icon should stay there and not flash outside.
> 
> **Screenshots** Since it flashes very fast I am still trying to show where it shows up ![image](https://user-images.githubusercontent.com/26686197/255686860-cbc8c26e-6bc9-48ab-af38-8e111520095c.png)
> 
> **please complete the following information:**
> 
>     * OS: Windows 10 Pro
> 
>     * Browser Firefox+Chrome (both extensions installed)
> 
>     * XDM addon Version XDM v8+ Browser Helper version 3.4 firefox
> 
>     * XDM Version  8.0.29
> 
> 
> **Generated log using below method**
> 
>     * https://github.com/subhra74/xdm/wiki/Generate-log-for-troubleshooting
> 
> 
> **Additional context** Add any other context about the problem here.

**Comment by inuonly** [comment]
I finally managed to get it recorded . Here is my diagnosis on the issue. 

I recorded system tray right when this issue was occuring and it occured for a long time giving me window to capture the event. 


I am attaching the screenshot showing the real time appearance of the event in system tray. Tried to attach video but there was no such option and it failed to attach video. 

![image](https://github.com/subhra74/xdm/assets/26686197/5e125307-d6bb-46ff-84c7-744693038a19)


I am also sharing the discection of the video in editor to show the frame and the frame itself as follows to show it is indeed xdm icon and it momentarily shows as expanded and within 1 frame it goes back to closeed tray. This keeps resizing icons on taskbar. 
![image](https://github.com/subhra74/xdm/assets/26686197/d9a2de92-3047-41e9-bc19-ba3ea2b91872)


Along with this at the time of recording i am also sharing the version adn screen shot showing it is updated to latest version available. 
![image](https://github.com/subhra74/xdm/assets/26686197/e7fef335-e8ff-4104-ace3-93041478dc2f)

![image](https://github.com/subhra74/xdm/assets/26686197/4c6fa9a6-35aa-4766-8f10-976ca8d932e0)
